shared/src/business/useCaseHelper/caseAssociation/associatePrivatePractitionerToCase.test.js

```diff
@@ -201,4 +201,59 @@ describe('associatePrivatePractitionerToCase', () => {
       serviceIndicator: SERVICE_INDICATOR_TYPES.SI_ELECTRONIC,
     });
   });
+
+  it('BUG 9323: should create log if practitioner is already associated with case but does not appear in the privatePractitioners array', async () => {
```

Comment:
just curious, why are we including "BUG 9323" in the log?

Reply:
Hmmm.  I think because we wanted to call attention to this bug, even in our test output.  Not a good answer, but an answer.  😅

Reply:
Okay thanks! My preference is not to include specific issue numbers in the code unless it's providing us some definitive value.

Reply:
That makes sense.  Are you OK to merge?  I, personally, don't think this is something to stop this merge.

Reply:
Yep! just a nitpick

Reply:
Since we are intentionally NOT fixing this bug, we felt it was best to highlight the issue as often as possible

Reply:
https://github.com/ustaxcourt/ef-cms/pull/1945#issuecomment-1059418145